### ISSUE TITLE
Shifted radius channel?

### ISSUE THREAD

**Issue by udion** [mkocabas/CoordConv-pytorch#13]
I went through the paper on archivX (this one https://arxiv.org/pdf/1807.03247.pdf)

but could not find the reason of why the center of the radius channel is shifted by 0.5 in both x and y direction? (Is there any particular reason? My intuition is it won't matter. Please let me know, if I'm missing something.)

**Comment by CharlesNord** [comment]
> I went through the paper on archivX (this one https://arxiv.org/pdf/1807.03247.pdf)
> 
> but could not find the reason of why the center of the radius channel is shifted by 0.5 in both x and y direction? (Is there any particular reason? My intuition is it won't matter. Please let me know, if I'm missing something.)

I think it's a mistake, since the author first normalized the channel coordinates to the interval [-1, 1], the center should be 0, not 0.5. 
It could be find from the original implementation in tensorflow, line 80 and line 81: https://github.com/uber-research/CoordConv/blob/master/CoordConv.py

**Comment by MaxwellRebo** [comment]
If it turns out this was a mistake, I may address it on my fork and do a PR. Did this ever get resolved?

**Comment by udion** [comment]
yeah I figured out it was a mistake.

**Comment by XZJIsme** [comment]
So, the conclusion is that the author of this repo made a mistake to shift the third channel with 0.5, right? @udion
